Open-Source/Contribute/Readme.md

```diff
@@ -22,6 +22,7 @@
 - [facebook-chat-api](https://github.com/Schmavery/facebook-chat-api)
 - [flexy](https://github.com/vladocar/flexy)
 - [GFOSS - Open Technology Alliance](https://summerofcode.withgoogle.com/archive/2021/organizations/5185060013080576)
+- [GitVio](https://github.com/devXprite/GitVio/)
```

Comment:
LGTM, thank you so much for contributing

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good